Jenkinsfile

```diff
@@ -93,7 +93,8 @@ pipeline {
               stash allowEmpty: true, name: 'build', useDefaultExcludes: false
               archiveArtifacts allowEmptyArchive: true,
                 artifacts: "${BASE_DIR}/elastic-apm-agent/target/elastic-apm-agent-*.jar,${BASE_DIR}/apm-agent-attach/target/apm-agent-attach-*.jar,\
-                      ${BASE_DIR}/apm-agent-attach-cli/target/apm-agent-attach-cli-*.jar,${BASE_DIR}/target/site/aggregate-third-party-report.html",
+                      ${BASE_DIR}/apm-agent-attach-cli/target/apm-agent-attach-cli-*.jar,${BASE_DIR}/apm-agent-api/target/apm-agent-api-*.jar,\
+                      ${BASE_DIR}/target/site/aggregate-third-party-report.html",
```

Comment:
While we are at it, maybe adding an extra link in the readme for the snapshot API would be relevant (https://github.com/elastic/apm-agent-java/#snapshots) We only list other artifacts there.

Reply:
Good idea, although we currently don't upload it, so we should handle separately